dashboard/app/views/layouts/application.html.haml

```diff
@@ -21,6 +21,7 @@
     = stylesheet_link_tag    'css/code-studio', media: 'all'
     = stylesheet_link_tag    'application', media: 'all'
     = render partial: 'layouts/analytics'
+    %script{src: webpack_asset_path('js/dcdo.js'), data: {'dcdo': DCDO.frontend_config.to_json}}
```

Comment:
I defer to @davidsbailey but am curious whether we could have this code load as part of an existing webpack bundle file to avoid another file request on every dashboard page load?

Reply:
Yeah that would be ideal. I was wondering if there is a javascript bundle we already use for both pegasus and dashboard.

Reply:
my question would be, why does it need to be separate bundle? or can this just be a js file in apps/src which gets pulled into whichever files you need it in via `import`?

Reply:
Worth noting that we don't include any bundle on most pegasus pages.  (There are just a few exceptions like `tutorialExplorer`.) 

That said, I know @maureensturgeon has been considering reimplementing the header's sign-in button & drop-down menu in React; if that work happened, we would certainly then have a bundle on every pegasus page.

Reply:
Fixed so that it does not add a new network request.